modules/update-release-images.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-release-images_{context}"]
+= Release images
+
+A release image is the delivery mechanism for a specific {product-title} (OCP) version.
+It contains the release metadata, a Cluster Version Operator (CVO) binary matching the release version, every manifest needed to deploy individual OpenShift Cluster Operators, and a list of SHA digest-versioned references to all container images that make up this OpenShift version.
+
+You can inspect the content of a specific release image by running the following command:
+
+[source,terminal]
+----
+$ oc adm release extract <release image>
+----
+
+.Example output
+[source,terminal]
+----
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.12.6-x86_64
+Extracted release payload from digest sha256:800d1e39d145664975a3bb7cbc6e674fbf78e3c45b5dde9ff2c5a11a8690c87b created at 2023-03-01T12:46:29Z
+
+$ ls
+0000_03_authorization-openshift_01_rolebindingrestriction.crd.yaml
+0000_03_config-operator_01_proxy.crd.yaml
+0000_03_marketplace-operator_01_operatorhub.crd.yaml
+0000_03_marketplace-operator_02_operatorhub.cr.yaml
+0000_03_quota-openshift_01_clusterresourcequota.crd.yaml <1>
+...
+0000_90_service-ca-operator_02_prometheusrolebinding.yaml <2>
+0000_90_service-ca-operator_03_servicemonitor.yaml
+0000_99_machine-api-operator_00_tombstones.yaml
+image-references <3>
+release-metadata
+----
+<1> Manifest for `ClusterResourceQuota` CRD, to be applied on Runlevel 03
```

Comment:
Do I know what Runlevel 03 is and who/how to apply the CRD?

Reply:
This is just meant to help annotate/describe what users are seeing in this example. Runlevels are described a few sections later on this page, but I don't think it's necessary to mention what a user might need to do with the CRD. It's just helpful context for users who happen to already know what that CRD might be.

(for context, this entire page is meant not for beginners, but for people already familiar with OCP who want to learn more about the technical details of an update)